src/libsystemd-network/sd-dhcp-lease.c

```diff
@@ -494,7 +494,7 @@ static int lease_parse_routes(
                 r = in4_addr_default_prefixlen((struct in_addr*) option, &route->dst_prefixlen);
                 if (r < 0) {
                         log_debug("Failed to determine destination prefix length from class based IP, ignoring");
```

Comment:
This should probably be deleted because `dhcp_lease_parse_options` complains about that as well